setup.py

```diff
@@ -15,12 +15,21 @@
     include_package_data=True,
     py_modules=['trie'],
     setup_requires=['setuptools-markdown'],
+    python_requires='>=2.7, !=3.5.2',
     install_requires=[
         "eth-hash>=0.1.0,<1.0.0",
         "eth-utils>=1.0.1,<2.0.0",
         "rlp>=0.4.7,<2.0.0",
         "cytoolz>=0.8.0,<1",
     ],
+    extras_require=[
+        "tox==2.6.0",
```

Comment:
Take a look here for how extras require should be structured:

https://github.com/ethereum/ethereum-python-project-template/blob/master/setup.py#L8-L29